webapp-django/settings_test.py

```diff
@@ -60,3 +60,9 @@
         'elasticsearch_urls': ['http://example:9123'],
     },
 }
+
+
+# Make sure we never actually hit a real URL when testing the
+# Crash-analysis monitoring.
+CRASH_ANALYSIS_URL = 'https://crashanalysis.m.c/something/'
+CRASH_ANALYSIS_HEALTH_DAYS = 2
```

Comment:
HEALTH_DAYS is a strange name. Consider a comment explaining this somewhere. I had to read the code to infer what it was doing.